Assets/AnkrSDK/Scripts/Ads/AnkrAds.cs

```diff
@@ -9,24 +11,95 @@ namespace AnkrSDK.Ads
 {
 	public static class AnkrAds
 	{
-		private const string PublicAddress = "public_address";
+		private const string DeviceIdFieldName = "device_id";
+		private const string AdTypeFieldName = "ad_type";
+		private const string AppIdFieldName = "app_id";
+		private const string PublicAddressFieldName = "public_address";
+		private const string LanguageFieldName = "language";
 
-		private const string RandomImageURLBase = "http://45.77.189.28:5001/ad";
+		private const string RandomImageURLBaseAD = "http://45.77.189.28:5001/ad";
+		private const string RandomImageURLBaseStart = "http://45.77.189.28:5001/start";
```

Comment:
Fix namings pls, 
not a RandomImageURLBase
but
AdServiceURLBase = "http://45.77.189.28:5001"
AdServiceURLRequestAd = $"{AdServiceURLBase}/ad";
AdServiceURLRequestStart = $"{AdServiceURLBase}/start";

Reply:
For this to work and retain a const we would need to use at least C# 10 version, is it okay if i set the language version to latest ?

Reply:
Actually even with the latest version, the IDE is ok with it but Unity still fires an Error.